src/utils/distance.js

```diff
@@ -0,0 +1,81 @@
+import dataService from '../services/data-service';
+import Util from './util';
+
+
+const Distance = {
+  unitConverter: {
+    m: 1,
+    km: 0.001,
+    mi: 0.000621371
+  },
+
+  distanceUnitsShort: {
+    meters: 'm',
+    kilometers: 'km',
+    miles: 'mi'
+  },
+
+
+  /**
+   * Gets name of current user's distance units
+   * @returns {string} name of current user's distance units
+   */
+  getUserDistanceUnit() {
+    return dataService.store.pilot.distanceUnit;
+  },
+
+  /**
+   * @returns {string} - short version of distance units
+   */
+  getUserDistanceUnitShort() {
+    return this.distanceUnitsShort[this.getUserDistanceUnit()];
+  },
+
+
+  /**
+   * Modifies distance in meters into current user's distance units
+   * @param {number|string} distance in meters
+   * @returns {number} distance in pilot's altitude units
+   */
+  getDistanceInPilotUnits(distance) {
+    const increment = this.unitConverter[this.getUserDistanceUnit()];
+    return parseFloat(distance) * increment;
+  },
```

Comment:
This method should accept distance in kilometers, not meters, since we will only have kilometers and miles for horizontal distances.

Reply:
I'd rather keep it as meters: I would return it as float for 'km' and js seems not to like multiple float/int conversions, so I think it is more proper (even when the result may not vary noticeably)